react/features/participants-pane/components/native/styles.js

```diff
@@ -195,17 +158,16 @@ export default {
 
     participantsPaneFooter: {
         alignItems: 'center',
-        backgroundColor: BaseTheme.palette.ui01,
+        backgroundColor: 'transparent',
```

Comment:
This my new design proposal. I attached a screen recording so you visually understand what I think would look much better, if we stick with full scroll.